src/plugins/lens_oss/kibana.json

```diff
@@ -0,0 +1,9 @@
+{
+    "id": "lensOSS",
```

Comment:
Can we please use `lensOss` and `mapsOss` to match `apmOss` and `securityOss`?

Reply:
Of course, will change this now

Reply:
Done